src/workers/vordrPostCommentPrevented.ts

```diff
@@ -0,0 +1,45 @@
+import { TypedWorker } from './worker';
+import { Comment } from '../entity';
+import { notifyNewVordrComment } from '../common';
+import { TypeORMQueryFailedError } from '../errors';
+import { logger } from '../logger';
+
+const worker: TypedWorker<'post-commented'> = {
+  subscription: 'api.vordr-post-comment-prevented',
+  handler: async ({ data }, con): Promise<void> => {
+    try {
+      const comment = await con
+        .getRepository(Comment)
+        .findOne({ where: { id: data.commentId }, relations: ['post'] });
+
+      if (!comment.flags.vordr) {
+        return;
+      }
+
+      const post = await comment.post;
+      if (post.private) {
+        return;
+      }
+      const user = await comment.user;
+
+      await notifyNewVordrComment(post, user, comment);
+      logger.info({ data }, 'Slack new comment message send');
```

Comment:
we don't need it i guess

Reply:
The logger?

Reply:
Probably not, we'll get it in slack anyways

Reply:
slack as a logger 😉